Store/Store/WebViewController/WebViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  WebViewController.swift
+//  Store
+//
+//  Created by Анна Сычева on 10.10.2022.
+//
+
+import UIKit
+import WebKit
+
+/// Экран вебвью
+final class WebViewController: UIViewController {
+    
+    // MARK: - Private visual Components
+    
+    private lazy var webView = WKWebView()
+    private lazy var toolBar = UIToolbar()
+    private lazy var backButtonItem = makeButtonItem(systemName: "chevron.left")
+    private lazy var forwardButtonItem = makeButtonItem(systemName: "chevron.right")
+    private lazy var spacer = UIBarButtonItem(systemItem: .flexibleSpace)
+    private lazy var refreshButtonItem = UIBarButtonItem(systemItem: .refresh)
+    private lazy var shareButtonItem = makeButtonItem(systemName: "square.and.arrow.up")
+    private lazy var progressView = UIProgressView(progressViewStyle: .default)
+    private lazy var progressViewItem = UIBarButtonItem(customView: progressView)
+    
+    // MARK: - Private properties
+    
+    private var observation: NSKeyValueObservation?
+    
+    // MARK: - Public properties
+    
+    var urlString = ""
+    
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .red
+
+        observation = webView.observe(\.estimatedProgress, options: [.new]) { _, _ in
```

Comment:
```suggestion
        observation = webView.observe(\.estimatedProgress, options: [.new]) { [weak self] _, _ in
```
https://stackoverflow.com/questions/49347331/is-weak-self-needed-in-nskeyvalueobservation-observe-closure